bot/seasons/halloween/8ball.py

```diff
@@ -0,0 +1,39 @@
+import asyncio
+import json
+import logging
+import random
+from pathlib import Path
+
+from discord.ext import commands
+
+log = logging.getLogger(__name__)
+
+with open(Path('bot', 'resources', 'halloween', 'responses.json'), 'r', encoding="utf8") as f:
+    responses = json.load(f)
+
+
+class EightBall(commands.Cog):
+    """Spooky Eightball answers."""
```

Comment:
make the class/methods strings more meaningful. Such as, what it is, what it is doing, how it is doing it, as example.

Reply:
Sorted.